packages/block-library/src/media-text/editor.scss

```diff
@@ -10,13 +11,24 @@
 		"resizer resizer";
 }
 
+.wp-block-media-text.is-vertically-aligned-top {
```

Comment:
It seems these styles duplicate what we have in styles.scss, I think they can be removed.